src/App.js

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
```

Comment:
Why is there a semi-colon here? Aren't you using es2015? Does react-scripts take care of this?